selfdrive/car/gm/radar_interface.py

```diff
@@ -38,7 +38,10 @@ class RadarInterface(RadarInterfaceBase):
   def __init__(self, CP):
     super().__init__(CP)
 
-    self.rcp = create_radar_can_parser(CP.carFingerprint)
+    if CP.radarOffCan:
+      self.rcp = None
+    else:
+      self.rcp = create_radar_can_parser(CP.carFingerprint)
```

Comment:
With this change, we can likely eliminate the logic on lines 19-20 as well

Reply:
Agree, just check radarOffCan like some of the other car ports do here